### DISCUSSION TITLE
{"message":"Missing/invalid DB name","status":500} Budibase Cloud

### DISCUSSION THREAD

**Discussion by aypm-dev** [Budibase/budibase#5895] [Help]
I updated my app to Budibase to v1.0.158 and users are getting a "Session not authenticated message" after they log into the application as public users.

Followed by it in the console appears a 403 error "{"message":"Missing/invalid DB name","status":500}" and the app doesn't load part of the forms fields, only the text, and the menu. Basically the same as #5573 and #5214 
The problem is that I'm not hosting my app using docker or any method of Self-hosting. I'm with the Budibase Cloud services. Publishing my application using the web page, so I can't find any way to interpret the solutions given in the discussions above. 

Help about how I can solve this will be appreciated!!

![image](https://user-images.githubusercontent.com/85044270/168452261-43aad182-9577-45e7-bb5a-397fa99feec1.png)

**Comment by aptkingston** [comment]
Hey @AngelCode-dev. What sort of data source are you using on this page? I've tried to reproduce this, including pulling data from a public access internal table, and it works fine - so I'm guessing it's something to do with the particular data source that's failing to load on this page.

**Reply by aypm-dev** [reply]
Yeah, I'm using data from a public internal table and pulling requests using the REST API. I changed all the tables and requests to public access and the problem is still there

**Reply by aypm-dev** [reply]
I'm going to debug if there's any specific element in the form causing the error
